internal/commands/secrets/delete_inputs.go

```diff
@@ -0,0 +1,83 @@
+package secrets
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const (
+	flagSecret      = "secret"
+	flagSecretShort = "s"
+	flagSecretUsage = "set the list of secrets to delete by ID or Name"
+)
+
+type deleteInputs struct {
+	cli.ProjectInputs
+	secrets []string
+}
+
+func (i *deleteInputs) Resolve(profile *cli.Profile, ui terminal.UI) error {
+	if err := i.ProjectInputs.Resolve(ui, profile.WorkingDirectory); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (i *deleteInputs) resolveSecrets(ui terminal.UI, allSecrets []realm.Secret) ([]realm.Secret, error) {
+	var toDelete []realm.Secret
+
+	if len(i.secrets) == 0 {
+
+		selectableSecrets := map[string]realm.Secret{}
+		selectableSecretOptions := make([]string, len(allSecrets))
+		for i, secret := range allSecrets {
+			option := displaySecretOption(secret)
+			selectableSecretOptions[i] = option
+			selectableSecrets[option] = secret
+		}
+		var selectedSecrets []string
+		askErr := ui.AskOne(
+			&selectedSecrets,
+			&survey.MultiSelect{
+				Message: "Which secret(s) would you like to delete?",
+				Options: selectableSecretOptions,
+			},
+		)
+
```

Comment:
[stylenit] omit this newline, also optionally rename this to `err` and/or inline the if-call